packages/vitest-pool-workers/package.json

```diff
@@ -66,13 +67,14 @@
 		"devalue": "^4.3.0",
 		"esbuild": "0.17.19",
 		"miniflare": "workspace:*",
+		"semver": "^7.5.1",
 		"wrangler": "workspace:*",
 		"zod": "^3.20.6"
 	},
 	"peerDependencies": {
-		"@vitest/runner": "1.3.0",
-		"@vitest/snapshot": "1.3.0",
-		"vitest": "1.3.0"
+		"@vitest/runner": "1.3.x - 1.5.x",
+		"@vitest/snapshot": "1.3.x - 1.5.x",
+		"vitest": "1.3.x - 1.5.x"
```

Comment:
TIL - hyphenated ranges!

Reply:
Same! `>=1.3.0 || <= 1.5.x` didn't work as I expected, because `1.2.0` satisfied this, as it's < 1.5.x, but hyphenated worked great!